### ISSUE TITLE
Option to have construction bot placed combinators off by default

### ISSUE THREAD

**Issue by kryojenik** [kryojenik/LTN_Combinator_Modernized#7]
Frequently, when copying and pasting certain elements in a factory, it is necessary to change some settings before starting things up. It would be nice to have a Per Player mod option to have pasted LTN Combinators be turned off by default so I don't have to hurry to try to grab them before LTN starts assigning trains.

**Comment by mrvn** [comment]
That would be something that needs to be turned on and off frequently. So maybe a button would be better than a mod option. When I copy an existing cell of my rail grid because I don't have enough of something then I want it to start up as soon as the bots place everything. But when I copy a station to build something new then I want the combinator off.

Maybe when you drop a blueprint on the button it could also turn the combinators off in the blueprint. If you need an example:  Landfill everything has a button where you drop a blueprint on it and it places landfill under all the entities in the blueprint so you can place it over water.

PS: Don't I wish one could edit ghosts of LTN Combinators...
